pipeline_api/src/main/java/com/splicemachine/pipeline/writehandler/UpdateUtils.java

```diff
@@ -9,9 +9,36 @@
 import com.splicemachine.storage.index.BitIndexing;
 import com.splicemachine.utils.ByteSlice;
 
-public abstract class AbstractWriteHandler implements WriteHandler {
+public class UpdateUtils {
 
-    protected KVPair getBaseUpdateMutation(KVPair mutation) {
+    public static KVPair deleteFromUpdate(KVPair mutation) {
```

Comment:
I think it makes sense to have a comment about the content of mutation passed in to this function. I feel the logic in this function that takes half of the fields can be confusing to someone who doesn't have the background of this problem. Just a suggestion.

Reply:
I added that explanation to `extractWriteFromUpdate` after I refactored `deleteFromUpdate` and `getBaseUpdateMutation` as @martinrupp suggested.